sys/include/net/netstats.h

```diff
@@ -41,6 +50,16 @@ typedef struct {
     uint32_t rx_bytes;          /**< received bytes */
 } netstats_t;
 
+/**
+ * @brief   Get a string description for @ref netstats_module_t, for debugging
```

Comment:
The description part of all three lines start at different indentations. I personally would like it more if you let them start at the same level.